src/reuse/comment.py

```diff
@@ -508,6 +509,15 @@ class ReStructedTextCommentStyle(CommentStyle):
     INDENT_AFTER_SINGLE = " "
 
 
+class RobotFrameworkCommentStyle(CommentStyle):
+    """RobotFramework comment style."""
+
+    SHORTHAND = "robot"
+
+    SINGLE_LINE = "#"
+    INDENT_AFTER_SINGLE = " "
+
+
```

Comment:
It looks to me like you could use `PythonCommentStyle` (among others) here. Please let me know if I'm wrong.